ffi/src/lib.rs

```diff
@@ -383,6 +383,7 @@ pub enum KernelError {
     FileAlreadyExists,
     MissingCommitInfo,
     UnsupportedError,
+    TableChangesDisabled,
```

Comment:
nit: feature/config is usually called 'change data feed'
```suggestion
    ChangeDataFeedUnsupported,
```